modules/apps/dynamic-data-mapping/dynamic-data-mapping-form-report-web/src/main/resources/META-INF/resources/js/components/list/List.es.js

```diff
@@ -38,7 +38,9 @@ export default ({data, field, totalEntries, type}) => {
 						<li key={index}>
 							{type == 'color' ? (
 								<Color hexColor={field} />
-							) : type == 'date' ? formatDate(field) : (
+							) : type == 'date' ? (
+								formatDate(field)
+							) : (
```

Comment:
@alinedoleron, why didn't you put this SF commit togeher with the previous one once both of them are just SF?

Reply:
madness >.< sorry. I'll not repeat this mistake.